pyat/at/tracking/patpass.py

```diff
@@ -43,7 +43,7 @@ def _atpass_one(ring, rin, **kwargs):
 
 
 def _atpass(ring, r_in, pool_size, globvar, **kwargs):
-    if platform.startswith('linux') and globvar:
+    if multiprocessing.get_start_method()=='fork':
         global globring
```

Comment:
Why did you remove globvar in the test ?

Reply:
it is a mistake